packages/extension-blockquote/package.json

```diff
@@ -15,14 +15,15 @@
   "type": "module",
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
+      "types": "./dist/packages/extension-blockquote/src/index.d.ts",
       "import": "./dist/index.js",
       "require": "./dist/index.cjs"
     }
   },
   "main": "dist/index.d.cjs",
```

Comment:
is it correct? not `"main": "dist/index.cjs",`

Reply:
Thanks, good find!